projects/samples/robotbenchmark/square_path/protos/SquarePathPioneer3dx.proto

```diff
@@ -4,7 +4,7 @@
 # The Pioneer 3-DX robot (Adept MobileRobots) is an all-purpose base, used for research and applications involving mapping, teleoperation, localization, monitoring, reconnaissance and other behaviors.
```

Comment:
There appears to be a difference compared to the master branch, something seems to be in front of the distance sensors
![a](https://user-images.githubusercontent.com/44834743/135426815-f1508ca1-05f6-4b4d-b791-a0082a9065a3.png)
![b](https://user-images.githubusercontent.com/44834743/135426823-34c41753-f5b3-41c8-b890-45d7c027d0c0.png)

I don't know if it's just a visual difference or if it actually impacts the sensors. Any clue what that might be?

Reply:
It was just a visual difference (problem with translations). It's fixed.